main/debugConsole.js

```
@@ -0,0 +1,113 @@
+  stickyBar.appendChild(closeBtn);
+  overlay.appendChild(stickyBar);
+
+  const append = (tag, args) => {
+    const line = document.createElement('div');
+    line.textContent =
+      `[${tag}] ` +
+      args
+        .map((a) => {
+          try {
+            return typeof a === 'string' ? a : JSON.stringify(a);
+          } catch (_) {
```

ESLint

[error] '_' is defined but never used.

(no-unused-vars)